README.rst

```diff
@@ -64,7 +64,7 @@ THE LATEST STABLE VERSION OF NeMo is **0.9.0** (which is available via PIP).
 **Requirements**
 
 1) Python 3.6 or 3.7
-2) PyTorch 1.2.* or 1.3.* with GPU support
+2) PyTorch 1.4* with GPU support
```

Comment:
I know I know - we are NVIDIA :)

But still, NeMo can work without GPUs, so can we say here: "(with or without GPU support)

Reply:
I think that is a decision you should ask Poonam or Negar to comment on